app/case-studies/[details]/page.tsx

```diff
@@ -10,15 +10,15 @@ import {
   CaseStudyTechStack,
   CaseStudyProblemStatement,
   CaseStudyChallengesAndLearnings,
-  CaseStudiesSimilarCaseStudies,
+  SimilarCaseStudies,
 } from "@/app/components/case-studies-components/index";
 interface CaseStudyProps {
   params: {
     details: string;
   };
 }
 
-const CaseStudy: React.FC<CaseStudyProps> = ({ params }) => {
+const CaseStudy: FC<CaseStudyProps> = ({ params }) => {
```

Comment:
Moving forward, try to extract your data from your params and it should prevent the error when you remove import react... Just a note for the future